src/layouts/LinkReport/components/LinkReportModal/LinkReportModal.tsx

```diff
@@ -0,0 +1,549 @@
+            fontFamily={typography.fontFamilies.inter}
+            overflow="hidden"
+            color={colors.base.content.default}
+            textOverflow="ellipsis"
+            alignSelf="flex-start"
+          >
+            {pageCmsUrl.split("/").at(-1)}
+          </Text>
+
+          <HStack
+            display="flex"
+            align-items="center"
+            gap="1.5rem"
+            width="100%"
+            paddingBottom="1.25rem"
+          >
+            <Text
+              fontSize="1.75rem"
+              fontStyle="normal"
+              fontWeight="600"
+              lineHeight="2.25rem"
+              fontFamily="Inter"
+              flex="1 0 0"
+              font-feature-settings="cv10 cv05"
+              alignSelf="flex-start"
+            >
+              {isBrokenLinksLoading
+                ? "Re-scanning page..."
+                : `${detailedErrorArr.length} broken references found`}
+            </Text>
+            <Link href={pageStagingUrl} isExternal>
+              <HStack spacing="1">
+                <Text>View page on staging</Text>
+              </HStack>
+            </Link>
+            <Link href={pageCmsUrl} isExternal>
+              <HStack spacing="1">
+                <Text>Edit page on CMS</Text>
+              </HStack>
+            </Link>
+          </HStack>
+          {!isBrokenLinksLoading && detailedErrorArr.length === 0 && (
+            <LinkReportModalNoBrokenLink {...props} />
+          )}
+
+          {(isBrokenLinksLoading || detailedErrorArr.length !== 0) && (
+            <>
+              <HStack
+                justifyContent="space-between"
+                display="flex"
+                width="100%"
+                paddingBottom="1.25rem"
+              >
+                <Text
+                  textStyle="body-2"
+                  fontFamily={typography.fontFamilies.inter}
+                  color="base.content.medium"
+                >
+                  After fixing the references, you can click the &quot;Run check
+                  for this page&quot; in the top right hand corner.
+                </Text>
+                <Box>
+                  <PaginateBtn
+                    currentPage={pageNum}
+                    totalPage={Math.max(
+                      1,
+                      Math.ceil(detailedErrorArr.length / 5)
+                    )}
+                    onPageChange={(newPage: number) => setPageNum(newPage)}
+                  />
+                </Box>
+              </HStack>
+              <TableContainer
+                mb="1.5rem"
+                width="100%"
+                backgroundColor="utility.ui"
+                borderRadius="8px"
+                border="1px"
+                borderColor="base.divider.medium"
+              >
+                <Table variant="simple">
+                  <Thead>
+                    <Tr>
+                      <Th textAlign="left" width="9rem" padding="0">
```

Comment:
Check width property. Seems to have some issues when the second column is too long